### DISCUSSION TITLE
Allow windows 11 shortucts in Simple Windows Switcher

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#1797] [Ideas]
Like Ctrl + w to close the window. Ability to customize them would also be nice

**Comment by MuscularPuky** [comment]
I need it too! it's supporting since win11, not win10

instead you can try Alt + Tab + Delete (both windows and sws)

**Comment by pyrates999** [comment]
Try Clavier+ for keyboard shortcut macros.
